src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutes.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsde.workbench.sam.api
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server
+import akka.http.scaladsl.server.Directives._
+import org.broadinstitute.dsde.workbench.sam.model._
+import org.broadinstitute.dsde.workbench.sam.service.UserService
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import org.broadinstitute.dsde.workbench.sam.model.SamJsonSupport._
+import spray.json.DefaultJsonProtocol._
+
+import scala.concurrent.ExecutionContext
+
+/**
+  * Created by mbemis on 5/22/17.
+  */
+trait UserRoutes extends UserInfoDirectives {
+  implicit val executionContext: ExecutionContext
+  val userService: UserService
+
+  def userRoutes: server.Route =
+    pathPrefix("user") {
+      requireUserInfo { userInfo =>
+        pathEndOrSingleSlash {
+          post {
+            complete {
+              userService.createUser(SamUser(userInfo.userId, Option(userInfo.userEmail))).map(user => StatusCodes.Created -> user)
```

Comment:
i know it's not new in this PR, but does `userEmail` still need to be an `Option`?